magento/pwa-studio
# import catalogActions to use when clearing filters

## Description

When clearing filters from the search page there was an error thrown in console because we forgot to import something (and we reference it directly in the file).

## Related Issue





Closes #ISSUE_NUMBER.

## Verification Steps

1. Go to the `/search.html?query=tops&category=292` page and open your console.
2. Click the `x` next to the category filter. You'll should see no console error.

You can do the same [here](https://develop.pwa-venia.com/search.html?query=tops&category=292) and see the error.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
